js/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@taplo/cli",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
I'm assuming those numbers are not expected to match those of `Cargo.toml`.

Reply:
They don't need to be matched.